### PR TITLE
Property definition tweaks

### PR DESCRIPTION
**Pull request by dan-gamble**
This adds an initial `Grid_Inside` variable until i can get around to making the postcss plugin that handles it.

Ref #79

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/helpers/property-definitions.css [hunk 1]
```diff
@@ -18,27 +18,29 @@
...
   --Grid_Gutter: 15px;
+  /* This differentiates to the Grid_Gutter in that this is used for the inside of panels and
+   cards. On all old projects this has been styled by using the Grid_Gutter. This is wrong, it
+   seemed right because the insides were normally 30ish and the Grid_Gutter was 30 as well so i
+   just assumed a natural connection between the 2. This actually has been coincidental and it's
+   just that we normally use a Grid_Gutter of 15 we haven't come across is */
+  --Grid_Inside: 15px;
 
   --GridContainer: {
     /* Add --Grid_Gutter to the container as the 'container' should be the size
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/helpers/property-definitions.css]: You may want to update this comment. :)

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/helpers/property-definitions.css]: What about it do you think needs updating?

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/helpers/property-definitions.css]: It says `Add --Grid_Gutter to the container as the 'container'` but that variable is not used anywhere in the definition.

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/helpers/property-definitions.css]: Oh sorry i thought you meant the big comment above the `G_I`

**Pull request merged**